meshtastic/tests/test_analysis.py

```
@@ -449,3 +443,162 @@ def test_analysis_camelcase_aliases_delegate() -> None:
+            """Simulate Dash app run method."""
+            _ = (debug, host, port)
+
+    def _fake_create_dash(*, slog_path: str) -> _FakeApp:
+        _ = slog_path
+        return _FakeApp()
+
+    monkeypatch.setattr(analysis_main, "create_dash", _fake_create_dash)
+    monkeypatch.setattr(
+        sys,
+        "argv",
+        ["fakescriptname", "--slog", os.devnull, "--debug", "--host", "0.0.0.0"],
```

Ruff (0.15.7)

[error] Possible binding to all interfaces

(S104)